packages/openapi3/src/util.ts

```diff
@@ -158,3 +159,36 @@ export function getDefaultValue(
 export function isBytesKeptRaw(program: Program, type: Type) {
   return type.kind === "Scalar" && type.name === "bytes" && getEncode(program, type) === undefined;
 }
+
+export function ensureValidComponentFixedFieldKey(
+  program: Program,
+  type: Type,
+  getDeclarationKey: () => string,
```

Comment:
why not just take the name and return the name

Reply:
Sorry, it's legecy code for trying to resolve conflict names afterward... please don't review, let me clean it up